ios/Classes/PostHogDisplaySurvey+Dict.swift

```diff
@@ -0,0 +1,104 @@
+#if os(iOS)
+    import Foundation
+    import PostHog
+
+    extension PostHogDisplaySurvey {
+        // Convert the survey object to a dictionary for communication with the Dart layer
+        // Native platform model -> Dictionary -> Dart model
+        func toDict() -> [String: Any] {
+            var dict: [String: Any] = [
+                "id": id,
```

Comment:
not a blocker but we could use constants for the key names coming directly from the iOS SDK
or the iOS class would have a builtin .toDict that is reused by this plugin, so the code lives in the iOS SDK instead

Reply:
that's correct, there will be duplication, but 2x better than 3x, your call here.